specification/redis/resource-manager/Microsoft.Cache/stable/2023-04-01/redis.json

```diff
@@ -2018,7 +2018,7 @@
           }
         },
         "publicNetworkAccess": {
-          "description": "Whether or not public endpoint access is allowed for this cache.  Value is optional but if passed in, must be 'Enabled' or 'Disabled'. If 'Disabled', private endpoints are the exclusive access method. Default value is 'Enabled'",
+          "description": "Whether or not public endpoint access is allowed for this cache.  Value is optional but if passed in, must be 'Enabled' or 'Disabled'. If 'Disabled', private endpoints are the exclusive access method. Default value is 'Enabled'. Note: this setting is important for caches with private endpoints; but has *no effect* on caches that are joined to, or injected into, a virtual network subnet.",
```

Comment:
Wanted to ensure if semi-colon was intentional.
`Note: this setting is important for caches with private endpoints; `
`Note: this setting is important for caches with private endpoints, `

Reply:
Good question. I'll run it by an editor!

Reply:
@flang-msft any suggestions to improve the dosc here? They eventually turn into this page here: 
https://learn.microsoft.com/en-us/rest/api/redis/redis/create?tabs=HTTP